README.md

```diff
@@ -82,11 +82,44 @@ def view_requiring_flag(request):
 def other_view(request):
     return HttpResponse('flag was not set')
 
-@flag_required('MY_FLAG', fallback_view=other_view)
+@flag_required('MY_FALLBACK_FLAG', fallback_view=other_view)
```

Comment:
Yes, I'm just trying to separate the two examples. You're right about the confusing name. `MY_FLAG_WITH_FALLBACK` it is!